app/controllers/foo_controller.rb

```diff
@@ -1,6 +1,8 @@
 class FooController < ActionController::Base  
   def new_user
     user = User.first(:conditions => ("username = '#{params[:username]}'"))
+    user = User.first(:conditions => ("username = '#{params[:username]}'"))
```

Comment:
__[staging][rubocop]__
* Lint/UselessAssignment: Useless assignment to variable - `user`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Style/RedundantParentheses: Don't use parentheses around a literal.

<!-- Comment by [SideCI](https://sideci.com) -->